### PR TITLE
task/move modules into modules directory and replace standard fns with lodash

### PR DESCRIPTION
**Pull request by jenniferarnesen**
Changes
* "module" files are moved utils directory
* icons.js is moved to an assets directory (anticipating that we will have more assets)
* replaced functions in old utils.js with lodash

### CODE REVIEW

**Comment by jenniferarnesen** [comment]
I'm still leaning toward the name "modules" over "utils" since I don't think everything in there qualifies as a utility.

**Comment by edoardo** [comment]
> * icons.js is moved to an assets directory (anticipating that we will have more assets)

I have a bunch of icons in the VisualizationTypeSelector, should we move them over to icons.js?

File: packages/app/src/actions/dimensions.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import { arrayToIdMap, sortArray } from '../util';
+import keyBy from 'lodash-es/keyBy';
+import { sortArray } from '../utils/util';
```
[the remark this answers is not in the document]
Comment (jenniferarnesen) [reply packages/app/src/actions/dimensions.js]: sortArray was doing the same thing as lodash sortBy

File: packages/app/src/components/Dimensions/DataDimension/DataDimension.js [hunk 2]
```diff
@@ -186,9 +186,13 @@ export class DataDimension extends Component {
...
 
-        const itemsToAdd = arrayToIdMap(
-            this.state.dimensionItems.filter(di => selectedIds.includes(di.id))
+        const itemsToAdd = keyBy(
+            this.state.dimensionItems.filter(di => selectedIds.includes(di.id)),
+            'id'
         );
+
+        console.log('itemsToAdd', itemsToAdd);
+
```
[the remark this answers is not in the document]
Comment (jenniferarnesen) [reply packages/app/src/components/Dimensions/DataDimension/DataDimension.js]: Sure, why not :)

File: packages/app/src/actions/dimensions.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (edoardo) [reply packages/app/src/actions/dimensions.js]: ok, so I don't understand why here we still use sortArray?

**Pull request merged**